### DISCUSSION TITLE
Any plans on adding slice?

### DISCUSSION THREAD

**Discussion by Masterofnoothing** [engageub/InternetIncome#62] [Ideas]
Slice has some popup ads feature and it works even when you are not using your browser 
like i left my pc running 1 night having my pc turned on with chrome running in the background and i made some money by morning without doing

**Comment by engageub** [comment]
Hi,
While there are many such extensions like addslice. It has to combined with ebesucher or adnade to save resources. But at the time, they have to support multi login from different locations. 
Do they allow login from different proxies at same time.
It pays only when you browse the websites. Having a standalone browser by installing and logging in may not get you rewards.
```
Simply install our browser extension. As you browse the internet, we’ll display ads on the pages you visit.
```

Thank you

**Reply by Masterofnoothing** [reply]
well thats outdated now  in their new version you see popup ads to earn too + yeah they do have multiple login but its not based on ip but on crome instances(diff acccts guest may work didnt try) the down side is you can link only 5 devices/month/account

**Comment by engageub** [comment]
Browser extensions links have been added in the list. They may or may not be added to Internet Income script based on the dependencies. 
https://github.com/engageub/InternetIncome

Thank you

**Comment by ppzzAn** [comment]
Hi
https://jetswap.com internet surfing potentially good earnings, maybe good plan to add on the future.

**Comment by engageub** [comment]
Hi,
Thank you for the link. Could you please let me know if you made withdrawals. Also note that only autosurfing will be supported if it is available. Manual surf will not be automated. The site has certification error when English language is selected. So I don't think I will consider this adding. 

Thank you
